react/features/chat/replacement.js

```diff
@@ -1,5 +1,22 @@
 import { regexes } from './smileys';
 
+/* eslint-disable no-useless-escape, max-len */
+const replacePatterns = {
+
+    // URLs starting with http://, https://, or ftp://
+    '<a href="$1" target="_blank" rel="noopener noreferrer">$1</a>':
+        /(\b(https?|ftp):\/\/[-A-Z0-9+&@#\/%?=~_|!:,.;]*[-A-Z0-9+&@#\/%=~_|])/gim,
+
+    // URLs starting with "www." (without // before it, or it'd re-link the ones done above).
```

Comment:
does this comment still apply, since we'd only be applying one pattern now?

Reply:
Actually no, will edit it.